README.md

```diff
@@ -50,8 +50,19 @@ to provide the best of both worlds.
         1. ***void fetchAd(AdRetriever retriever, RetrieveAdCallback callback)*** 
         2. ***void fetchQuePoint(CuePointsRetriever retriever, CuePointCallBack callBack)***
         
-
-
+    5. If you want to quickly test the pre-roll and mid-roll video ads using the default components, just simply change 1 line of code in [PlayerModuleDefault](lib/src/main/java/com/tubitv/media/di/PlayerModuleDefault.java#L131): <br/>  
```

Comment:
Nitpick, but I've heard it's recommended in writing to write single digits in text, so "one" instead of "1".  Might be worth changing, but your call.

Reply:
i thought "1" might be easy to see, and emphasize the easiness of change .

Reply:
Ok, sounds good!